omod/src/main/webapp/fragments/decorator/standardEmrPage.gsp

```diff
@@ -14,6 +14,9 @@
         <title>${ title ?: "OpenMRS" }</title>
         <link rel="shortcut icon" type="image/ico" href="/${ ui.contextPath() }/images/openmrs-favicon.ico"/>
         <link rel="icon" type="image/png\" href="/${ ui.contextPath() }/images/openmrs-favicon.png"/>
+        <!-- Latest compiled and minified CSS -->
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
```

Comment:
Will this work without an internet connection?

Reply:
Hi @dkayiwa  nope it will not.Are we having situations where the ref app needs to work without internet? If so we can  have bootstrap downloaded and have it.

Reply:
In the developing world, you cannot assume presence of internet.

Reply:
Sure will be adding the local reference for bootstrap.

Reply:
@ayesh did you try finding out the cause of travis ci failure?

Reply:
you don't need to have the whole bootstrap for this usage, you can generate minimized bootstrap with only used components here.

Reply:
Hi @suthagar23 what do u mean by minimized  bootstrap for the components.This page is like the common page for all other sub pages.Like in angular apps index.html.So this integration would affect other pages.